packages/xo-web/src/common/xo/choose-sr-for-each-vdis-modal/index.js

```diff
@@ -85,7 +86,10 @@ export default class ChooseSrForEachVdisModal extends Component {
               </SingleLineRow>
               {map(props.vdis, vdi => (
                 <SingleLineRow key={vdi.uuid}>
-                  <Col size={6}>{vdi.name_label || vdi.name}</Col>
+                  <Col size={6}>
+                    {vdi.name_label || vdi.name}{' '}
+                    <span className='text-muted'>({formatSize(vdi.size)})</span>
```

Comment:
You can use [`Vdi` from `render-xo-item`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/render-xo-item.js#L273-L309) and add a `showSize` prop to it.